addresses/base.json

```diff
@@ -0,0 +1,574 @@
+      "address": "0x0a9b2080C14DaF008AE87C977dFDf5f5D6D0937F",
+      "id": "0xa267eb4d",
+      "path": "contracts/core/strategy/BotAuth.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "RecipeExecutor",
+      "address": "0xdDFFd19564F9703800Da8a2FB9c2b4a7242bf01F",
+      "id": "0x4c69ee1e",
+      "path": "contracts/core/RecipeExecutor.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "DFSProxyRegistryController",
+      "address": "0x50bCFC115283dF48Ab6382551B9B93b08E197747",
+      "id": "0xcbbb53f2",
+      "path": "contracts/utils/DFSProxyRegistryController.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "PullToken",
+      "address": "0x5B0B7E38C2a8e46CfAe13c360BC5927570BeEe94",
+      "id": "0xcc063de4",
+      "path": "contracts/actions/utils/PullToken.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "WrapEth",
+      "address": "0x491cc4AFbE0081C3464DeF1114ba27BE114b2401",
+      "id": "0x11135183",
+      "path": "contracts/actions/utils/WrapEth.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "UnwrapEth",
+      "address": "0xcF91546046F16B3c38b890CC508E280BEffa66b9",
+      "id": "0x929145d0",
+      "path": "contracts/actions/utils/UnwrapEth.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "SendToken",
+      "address": "0x1420f4977E7B71AFddccBFc6F6e1505CefdF99F0",
+      "id": "0x02abc227",
+      "path": "contracts/actions/utils/SendToken.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "SendTokenAndUnwrap",
+      "address": "0xF32d5d8D81f2662A02955CE537537088DF29daf5",
+      "id": "0x17782156",
+      "path": "contracts/actions/utils/SendTokenAndUnwrap.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "SendTokens",
+      "address": "0x17B4ecd173b3Df2F5cB02c53Df8AA34e23Bcb92E",
+      "id": "0xa87d9d0e",
+      "path": "contracts/actions/utils/SendTokens.sol",
+      "version": "1.0.0",
+      "inRegistry": true,
+      "changeTime": "86400",
+      "registryIds": [],
+      "history": []
+    },
+    {
+      "name": "FLAaveV3",
+      "address": "0x04ce4b2a9F524d976a8eD8a49B9313C5a2C3ccAD",
```

Comment:
Can we deploy FLAaveV3WithFee, and then send this 0x04 address to Aave governance to lose the fee?
Also FLAction could be deployed too